### PR TITLE
Added the animals table

### PR DESCRIPTION
**Pull request by andersonlebon**
#### Here are the following tasks that we tried to manage in this PR:

- [ ] I created a database
- [ ] I created a table
- [ ] I populated the table with data
- [ ] I queried the database

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('REX', '03-02-2020', 0, TRUE, 20);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Saggapus', '15-11-2020', 12, TRUE, 8);
+INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Pikachu', '7-01-2021', 10, FALSE, 19.04);
+INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Milu', '12-05-2017', 7, TRUE, 14.10);
```
Comment (arslanbisharat) [inline data.sql]: - [ ] Let's add correct details in your data.sql as requested in the project [guidelines](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_create_tables.md). 
Capture

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+
+SELECT date_of_birth FROM animals WHERE name='Saggapus' OR name='rex';
+
+SELECT name, escape_attempts FROM animals WHERE weight_kg>8;
+
+SELECT * FROM animals WHERE neutered=true;
+
+SELECT * FROM animals WHERE NOT name='Milu';
+
+SELECT * FROM animals WHERE weight_kg BETWEEN 10 AND 20;
```
Comment (arslanbisharat) [inline queries.sql]: - [ ] Let's make sure you are following the [guidelines](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_create_tables.md) properly. 
Capture

**Review by arslanbisharat — COMMENTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashbaord. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Comment by andersonlebon** [comment]
Thanks very much for your kind review

**Pull request merged**